cmd/repo-init/main.go

```diff
@@ -211,19 +212,27 @@ A test named %s already exists. Please choose a different name.\n`, test.As)
 					break
 				}
 			}
+
+			clusterProfiles := sets.NewString("gcp", "aws", "azure4")
```

Comment:
I think it would be user-friendly to hide the `azure4` quirk from the user (let them select `azure` and map the value to `azure4` internally)